docs/cmake-integration.md

```diff
@@ -83,6 +83,8 @@ catch_discover_tests(target
                      [TEST_SUFFIX suffix]
                      [PROPERTIES name1 value1...]
                      [TEST_LIST var]
+                     [REPORTER reporter]
+                     [OUTPUT_DIR dir]
 )
```

Comment:
This is missing the prefix and suffix args.

Reply:
Fixed, thanks!